integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/HyperfoilResource.java

```diff
@@ -66,6 +68,17 @@ public static boolean containsFailedRunPhase(String idRun) {
         return run.getPhases().stream().anyMatch(Phase::getFailed);
     }
 
+    public static void storeBenchmarkReport(String perfTestName, String runId) {
+        try {
+            String generateReport = validation.generateReport(validation.getRun(runId));
+            Path destination = Paths.get(BENCHMARK_REPORT_DESTINATION_FOLDER_PATH, perfTestName + "-" + LocalDateTime.now() + ".html");
+            Files.createDirectories(destination.getParent());
+            Files.writeString(destination, generateReport);
+        } catch (IOException e) {
+            throw new RuntimeException("error while generating benchmark report", e);
+        }
```

Comment:
do we really need to stop the execution if unable to store this file locally? IMHO if the test is not executed locally, I think we should skip this exception, and just log a warning - wdyt?